src/GitExtensions.PluginTemplate/Plugin.cs

```diff
@@ -43,14 +43,21 @@ public override void Unregister(IGitUICommands gitUiCommands)
             
         }
 
+        ///// <summary>
+        ///// See: https://github.com/gitextensions/gitextensions.plugintemplate/wiki/GitPluginBase#public-virtual-ienumerableisetting-getsettings
+        ///// </summary>

```

Comment:
💭  Not specific to this PR, but I don't think it is a good idea to put links in the summary on their own.
The summary should describe the purpose of each member, so it comes up in intellisense.

Reply:
I'm not very fond of too because if you're new, that painfull and you will click endlessly on it.
But that was like that, so I continue like that to prevent the painfull task to sync the doc here and in the wiki.

Reply:
You're both right. It would be much easier to access the content through intellisense, not through some external wiki page.

Reply:
>The summary should describe the purpose of each member, so it comes up in intellisense.

I don't get you exactly, for the intellisense part (even if we all agree that the links should be replaced by the documentation text).

**The `<summary>` content is here mainly to instruct the plugin developer what he should put inside the body of the methods and not to document for callers of the api.**

So it won't be displayed in any intellisense.

And if we add the comments in the `GitPluginBase.cs` (like I started to do it in the PR #6888: https://github.com/gitextensions/gitextensions/pull/6888/files#diff-cc0458fac23c6fb4ef0646008a15be1c ) from which this class is inherited, I think they won't show in the intellisense either because the reference is made as a dll import on a dll built in release (from the downloaded portable released version of GitExtensions).

Reply:
Yes, you're absolutely right...

In this case I suggest we remove xml-doc tags, as they are primarily intended for intellisense and docs generation.
Anything you wish to communicate to developers should probably be presented in forms of comments

Reply:
Done.